src/main/java/org/prebid/server/analytics/reporter/agma/AgmaAnalyticsReporter.java

```diff
@@ -116,6 +116,11 @@ public <T> Future<Void> processEvent(T event) {
         }
 
         final AuctionContext auctionContext = contextAndType.getLeft();
+        final String eventType = contextAndType.getRight();
+        if (auctionContext == null) {
+            return Future.succeededFuture();
+        }
```

Comment:
I'm not sure I know the case when auctionContext is null, because it should always be created for each auction

What's the case you've face to make this check?

Reply:
Sorry, I've skipped the description)

Yeah, I see. Interesting.